### ISSUE TITLE
Still seems we have problems when updating data into the metadata configmap

### ISSUE THREAD

**Issue by lepiolet** [appgate/sdp-operator#58]
We keep treating this entity like modified because the generation is not updated in the metadata configmap. 

``` 
2020-09-24 08:00:45,225 [INFO] [appgate-state] Validating expected state entities
2020-09-24 08:00:45,805 [INFO] [appgate-operator/appgate-test] Updating current state from controller
2020-09-24 08:00:48,183 [INFO] [appgate-operator/appgate-test] No more events for a while, creating a plan
2020-09-24 08:00:48,728 [INFO] [appgate-operator/appgate-test] AppgatePlan Summary:
2020-09-24 08:00:48,728 [INFO] [appgate-operator/appgate-test] * <class 'appgate.openapi.parser.GlobalSettings'> GlobalSettings [GlobalSettings]
2020-09-24 08:00:48,728 [INFO] [appgate-operator/appgate-test]    DIFF for GlobalSettings:
2020-09-24 08:00:48,728 [INFO] ---
2020-09-24 08:00:48,728 [INFO] +++
2020-09-24 08:00:48,728 [INFO] @@ -10,3 +10,3 @@
2020-09-24 08:00:48,728 [INFO]      "appDiscoveryDomains": [],
2020-09-24 08:00:48,729 [INFO] -    "generation": 1
2020-09-24 08:00:48,729 [INFO] +    "generation": 2
2020-09-24 08:00:48,729 [INFO]  }
```

**Comment by lepiolet** [comment]
After restarting the pod the problem is gone, the problem is that the entity in memory does not update the metadata so the generation in memory (n) it's always lower than  the latest generation (n+1)

**Comment by lepiolet** [comment]
Fixed in #61
